scripts/submit_eval_jobs.py

```diff
@@ -69,34 +69,23 @@ def adjust_gpus(task_spec, experiment_group, model_name, gpu_multiplier):
 today = date.today().strftime("%m%d%Y")
 
 parser = argparse.ArgumentParser()
-parser.add_argument("--workspace", type=str, default="hamishivi")
+parser.add_argument("--workspace", type=str, default="oe-adapt-general")
 parser.add_argument("--model_name", type=str, default="hf-opt-7B")
 parser.add_argument("--location", type=str, default=None)
 parser.add_argument("--beaker_image", type=str, default=None, help="If given, use this Beaker image.")
 parser.add_argument("--beaker_subfolder", type=str, default=None)
-parser.add_argument("--cluster", nargs='+', default=["ai2/allennlp-cirrascale", "ai2/general-cirrascale", "ai2/general-cirrascale-a100-80g-ib", "ai2/mosaic-cirrascale-a100", "ai2/s2-cirrascale-l40"])
+parser.add_argument("--cluster", nargs='+', default=["ai2/allennlp-cirrascale", "ai2/general-cirrascale", "ai2/general-cirrascale-a100-80g-ib", "ai2/mosaic-cirrascale-a100", "ai2/s2-cirrascale-l40", "ai2/jupiter-cirrascale-2"])  # nb: jupiter requires no nfs models.
```

Comment:
I have tried it - is downloading of beaker dataset fast on jupiter? If not, maybe we can exclude the jupiter cluster here.

Reply:
Oh good point. I dug through my older evals and found this job which ran on jupiter-2:
https://beaker.org/ex/01J1X8AX4ZK9ZNXWTRZAY0NG1P/tasks/01J1X8AX6VMYW8CXJGE07E5CPQ/job/01J1X8B4547BZQSSPRQKCH0WGR 
It took ~20min to init, while other jobs took ~5min, so I guess it is much slower. I'll remove it from the default list.